rosdep/python.yaml

```diff
@@ -7018,6 +7018,9 @@ python3-websocket:
   openembedded: ['python3-websockets@meta-python']
   rhel: ['python%{python3_pkgversion}-websocket-client']
   ubuntu: [python3-websocket]
+python3-websockets:
+  debian: [python3-websockets]
+  ubuntu: [python3-websockets]
```

Comment:
Please add entries for 

- arch: https://archlinux.org/packages/community/x86_64/python-websockets/
- fedora: https://src.fedoraproject.org/rpms/python-websockets
- gentoo: https://packages.gentoo.org/packages/dev-python/websockets

Reply:
Please add openembedded ['python3-websockets@meta-python'] as well https://layers.openembedded.org/layerindex/recipe/82184/

Reply:
Both done.